### ISSUE TITLE
TreeSelect: Clear Icon Misplacement

### ISSUE THREAD

**Issue by chiefdeals** [primefaces/primereact#5020]
### Describe the bug

Misalignment on 'clear icon' for TreeSelect component - on LTR aligns to the far left with no padding. Expected, align right, padding/margin right

Seen in documentation

Safari, Firefox, Chrome - versions irrelevant as they're all identical

### Reproducer

https://primereact.org/treeselect/

### PrimeReact version

current

### React version

18.x

### Language

TypeScript

### Build / Runtime

Next.js

### Browser(s)

 Safari, Firefox, Chrome - versions irrelevant as they're all identical

### Steps to reproduce the behavior

go to https://primereact.org/treeselect/

go to clear icon

derp

### Expected behavior

Misalignment on 'clear icon' for TreeSelect component - on LTR aligns to the far left with no padding. Expected, align right, padding/margin right

**Comment by melloware** [comment]
Duplicate of https://github.com/primefaces/primereact/issues/5012

Which is waiting on this fix: https://github.com/primefaces/primereact-sass-theme/pull/32

**Comment by chiefdeals** [comment]
Apologies for duplicate, kinda new to this world.
